APIs/gettyimages.com/patch.yaml

```diff
@@ -4,3 +4,10 @@ info:
   x-logo:
     backgroundColor: '#333333'
     url: 'http://www.gettyimages.com/landing/assets/unisporkal_engine/images/getty-logo-negative-b117d413642b91fba020901fbd8949ba.svg'
+securityDefinitions:
+  api_key:
+    in: header
+    name: api_key
+    type: apiKey
+security:
+  - api_key: []
```

Comment:
@maryguirola Seet this red mark, it's because your editor doesn't add a new line at the end of the file.
http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file